frontend/src/components/VaultDetails.vue

```diff
@@ -260,14 +268,17 @@ async function fetchData() {
 
 async function fetchOwnerData() {
   try {
-    const vaultKeyJwe = await backend.vaults.accessToken(props.vaultId, true);
-    vaultKeys.value = await loadVaultKeys(vaultKeyJwe);
     (await backend.vaults.getMembers(props.vaultId)).forEach(member => members.value.set(member.id, member));
     usersRequiringAccessGrant.value = await backend.vaults.getUsersRequiringAccessGrant(props.vaultId);
     vaultRecoveryRequired.value = false;
+    const vaultKeyJwe = await backend.vaults.accessToken(props.vaultId, true);
+    vaultKeys.value = await loadVaultKeys(vaultKeyJwe);
+    paymentRequired.value = false;
```

Comment:
Because it can change over time. If the vault owner removes a member, the license may not be exceeded anymore. If we did not override this variable, it would always stay at `paymentRequired`, see https://github.com/cryptomator/hub/pull/258/commits/93141840df03393b87500c622414b75d2eb9337c